packages/host/tests/helpers/index.gts

```diff
@@ -206,6 +185,22 @@ class MockMessageService extends Service {
   }
 }
 
+export function setupOnSave(
```

Comment:
Maybe we should consider calling this in in our `setupApplicationTest` and `setupRenderingTest` and make it available automatically in all tests as `this.onSave(...)`?

Reply:
on challenge with this is that the `TestContext` type does not have an `onSave()` method on it. we can certainly slap our own functions on to the `TestContext`, but we'll get type errors trying to use it unless we cast to any. so type wise this is a little awkward...